index.js

```diff
@@ -5,7 +5,7 @@ module.exports.Util = require('./lib/util');
 module.exports.ZwaveDevice = require('./lib/zwave/ZwaveDevice.js');
 module.exports.ZwaveLockDevice = require('./lib/zwave/ZwaveLockDevice.js');
 module.exports.ZwaveMeteringDevice = require('./lib/zwave/ZwaveMeteringDevice.js');
-module.exports.ZwaveXYLightDevice = require('./lib/zwave/ZwaveXYLightDevice.js');
+module.exports.ZwaveXYLightDevice = require('./lib/zwave/ZwaveLightDevice.js');
```

Comment:
did you also want to change the `module.exports.ZwaveXYLightDevice`? or just the `.js`

Reply:
Also the export, something went wrong it seems, thanks!